Banking/src/main/java/com/example/Banking/service/AccountService.java

```diff
@@ -0,0 +1,32 @@
+package com.example.Banking.service;
+import com.example.Banking.exception.IdNotFoundException;
+import com.example.Banking.model.Account;
+import org.springframework.beans.factory.annotation.Autowired;
+import java.util.List;
+
+@org.springframework.stereotype.Service
+public class AccountService {
+
+    @Autowired
+    private AccountDao accountDao;
+
+    public List<Account> getEveryAccount() {
+        return accountDao.getAll();
+    }
+
+    public Account getAccountbyId(Long acNumber) throws IdNotFoundException {
+        return accountDao.getAccount(acNumber);
+    }
+
+    public void createAccount(Account account) {
+         accountDao.insertAccount(account);
+    }
+
+    public void deleteAccountbyId(Long acNumber) throws IdNotFoundException {
+        accountDao.deleteAccount(acNumber);
+    }
+
+    public void updateAccountbyId(Account account) throws IdNotFoundException {
```

Comment:
Now since we are not updating on Id the method should be renamed to updateAccount

Reply:
I have renamed to updateAccount and pushed it